content/general/authentication.textile

```diff
@@ -226,176 +235,59 @@ auth.requestToken({ capability: {
-- nonce := an unquoted, un-escaped random string of at least 16 characters.
+See a working "capabilities example":<%= JsBins.url_for('authentication/capabilities') %>.
 
 h3(#capability-operations). Capability operations
 
-The following capability operations are available for API keys and issued tokens.
+<%= partial 'general/_authentication_capabilities' %>
 
-- subscribe := can subscribe to messages and presence state change messages on channels
-- publish := can publish messages to channels
-- presence := can register presence on a channel (enter, update and leave)
-- history := can retrieve message and presence state history on channels
-- stats := can retrieve current and historical usage statistics for an app
+h3(#identified-clients). Understanding Identified clients
 
-See a working "capabilities example":<%= JsBins.url_for('authentication/capabilities') %>, read "understanding capabilities and token security":#capabilities-explained above to get a more thorough overview of how capabilities can be used to secure your application along with working examples.
+When a client is authenticated and connected to Ably, they are considered to be an *authenticated client*. However, whilst an *authenticated client* has a verifiable means to authenticate with Ably, they do not necessarily have an identity. When a client is assigned a trusted identity (i.e. a @client ID@), then they are considered to be an *identified client* and for all operations they perform with the Ably service, their @client ID@ field will be automatically populated and can be trusted by other clients.
 
-h3. HMAC calculation
+For example, assuming you were building a chat application and wanted to allow clients to publish messages and be present on a channel. If each client is assigned a trusted identity by your server, such as a unique email address or UUID, then all other subscribed clients can trust any messages or presence events they receive in the channel as being from that client. No other clients are permitted to assume a @client ID@ that they are not assigned in their token or token request, that is they are unable to masquerade as another @client ID@.
 
-First the canonicalized request text, constructed as follows:
+In Ably a client can be identified with a @client ID@ in three ways:
 
-* start with the empty string
-* for each of the following fields in order: key ID, ttl, capabilities, clientId, timestamp, nonce, even when empty
-** append the canonicalized string value for that field
-** append a newline (0xa) character.
-Note that a newline character is added for each field, including any empty client ID string, and the last (nonce) field.
+* if the client is authenticated with a token that is issued for that @client ID@;
+* if the client claims that @client ID@ and is authenticated using "Basic Authentication":#basic-authentication (using an API key);
```

Comment:
I thought basic auth clients were never identified? At least for the purposes of the 'only allow identified clients' namespace rule (of course, they can still publish / enter presence with a clientId)